### DISCUSSION TITLE
Request clean build

### DISCUSSION THREAD

**Discussion by WAR-DEPT** [ROM-builders/temporary#23904] [General]
It is successfully synced though I got this error, it had happened many times with me.

https://cirrus-ci.com/task/6282078451400704

**Comment by Apon77** [comment]
![image](https://github.com/ROM-builders/temporary/assets/61594531/d80e0114-429b-4c70-a3f8-dd607d5ccc66)

Clean build is showing this.
Also, I don't see any update in bananadroid source recently. Either its from your side, or from bananadroid side. We cant do anything in this situation.
